JavaScript/JavaScriptSDK/Contracts/Generated/ContextTagKeys.ts

```diff
@@ -35,20 +38,38 @@ module AI
         public userAgent: string;
         public userId: string;
         public userStoreRegion: string;
+        public userAuthUserId: string;
+        public userAnonymousUserAcquisitionDate: string;
+        public userAuthenticatedUserAcquisitionDate: string;
         public sampleRate: string;
         public internalSdkVersion: string;
         public internalAgentVersion: string;
+        public internalDataCollectorReceivedTime: string;
```

Comment:
I agree the internals have no purpose here - but this file is auto-generated. Consulting with Max we thought we should leave them as is - I dont think it's harmful. 

But if you think otherwise I will manually remove them..